AsyncDisplayKit/ASVideoNode.h

```diff
@@ -0,0 +1,34 @@
+
+#import <AsyncDisplayKit/AsyncDisplayKit.h>
+
+typedef NS_ENUM(NSUInteger, ASVideoGravity) {
+  ASVideoGravityResizeAspect,
+  ASVideoGravityResizeAspectFill,
+  ASVideoGravityResize
+};
+
+@protocol ASVideoNodeDelegate;
+
+@interface ASVideoNode : ASControlNode<_ASDisplayLayerDelegate>
+@property (atomic, strong, readwrite) AVAsset *asset;
+@property (atomic, strong, readonly) AVPlayer *player;
+@property (atomic, strong, readonly) AVPlayerItem *currentItem;
+@property (nonatomic, assign, readwrite) BOOL shouldAutoplay;
+@property (atomic) ASVideoGravity gravity;
+@property (atomic) BOOL autorepeat;
```

Comment:
later we should add support for autoreverse, too :)  I suppose the behavior would be to play once forward and once back, unless autorepeat is on too, in which case it would continue.